frontend/src/constants/toolbarItems.ts

```diff
@@ -0,0 +1,28 @@
+import { ToolbarItem } from "../types";
```

Comment:
It's okay to define toolBarItems! This idea is great! Although this is good, are these constants used by any other components without Drawermenu? If not, these constants should be included in Drawermenu. Because this means this is dedicated to Drawermenu.

Reply:
Okay. I moved the constants in Drawer Contents.